src/features/duckplayer/overlays.js

```diff
@@ -357,8 +131,23 @@ export class Environment {
         this.debug = Boolean(params.debug)
     }
 
+    /**
+     * This is the URL of the page that the user is currently on
+     * It's abstracted so that we can mock it in tests
+     * @return {string}
+     */
     getPlayerPageHref () {
         if (this.debug) {
```

Comment:
File a follow up task to remove this, it shouldn't be in production code at all.

Reply:
done